### ISSUE TITLE
Remoting issue (I think)

### ISSUE THREAD

**Issue by shukenmg** [microsoft/Microsoft-OpenXR-Unreal#61]
When I am remoting I get this error from time to time, it doesn't crash the editor, but it makes debugging code a pain because it keeps hitting a breakpoint.

```
LogOutputDevice: Error: Ensure condition failed: CurrentGCObject->GetReferencerName() != FGCObject::UnknownGCObjectName [File:D:\build\++UE5\Sync\Engine\Source\Runtime\CoreUObject\Private\Misc\GCObjectReferencer.cpp] [Line: 166] 
Please make sure all FGCObject derived classes have a unique name by overriding FGCObject::GetReferencerName() function. FGCObject::GetReferencerName() will become pure virtual in the next engine release. See callstack for details.
```

![image](https://user-images.githubusercontent.com/18000205/168816867-2410e1be-cd4e-4c78-81df-2206a704cca6.png)
![image](https://user-images.githubusercontent.com/18000205/168816895-9e590e28-381d-4f26-984a-5f21ed9b6f49.png)

Thanks!

**Comment by fieldsJacksonG** [comment]
Fixed in 1.1.13 - marketplace is currently updating to this release

**Comment by shukenmg** [comment]
Fixed, thanks!
